src/submodules/io/csv_integration.jl

```diff
@@ -63,6 +100,33 @@ function populate_from_csv!(
     end
 end
 
+"""
+    function dump_to_csv(filename, agents, fields; kwargs...)
```

Comment:
Since these functions are not accessible in direct scope, their docstring should be prefaced with `ModelIO.`